.travis.yml

```diff
@@ -26,7 +27,6 @@ matrix:
 
 cache:
   directories:
-  - $GOPATH/src
```

Comment:
:sweat_smile: I'm surprised the build directory is inside this path but hey, nice catch!